streams/src/main/java/org/apache/kafka/streams/processor/internals/StateDirectory.java

```diff
@@ -681,4 +701,18 @@ public int hashCode() {
         }
     }
 
+    public static class BackoffRecord {
+        private long attempts;
+        private long lastAttemptMs;
+        private static final ExponentialBackoff EXPONENTIAL_BACKOFF = new ExponentialBackoff(1, 2, 1000, 0.5);
```

Comment:
`MaxInterval` is set to `1000`. Should it be increased?

Reply:
Agreed, once per second still seems like a lot. We could start with 10 seconds, and see if we want to increase it. wdyt, @cadonna ?